Packs/IronscalesEventCollector/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Ironscales Event Collector
+
+- Added new **Collect all incidents** parameter, upon checking it all incidents will be fetched instead of open ones only.
```

Comment:
```suggestion
Added the new **Collect all incidents** parameter. When checked, it fetches all issues instead of just open ones.
```
incidents?alerts?issues?events?